content/lxd/getting-started-cli.md

```diff
@@ -143,13 +143,13 @@ Instructions on building and installing LXD from source [can be found here](http
 
 Before you can create an instance, you need to configure LXD.
 
-Run the following command as root to specify the configuration options:
+Run the following command to specify the configuration options:
 
-    lxd init
+    sudo lxd init
```

Comment:
Will this then work for all distros?

Reply:
I think most distributions (at least for full desktop environments or cloud images) will have sudo present.
When they don't, it tends to be easy for the user to either find instructions to install it or to manually run things as root.

Reply:
@monstermunchkin could you make sure all our generated images include `sudo` so that our instructions at least work with all our own stuff? :)

Reply:
https://github.com/lxc/lxc-ci/pull/372